cypress/support/commands.ts

```diff
@@ -297,6 +297,11 @@ export const upsertSecret = (
   })
 }
 
+// TODO: (sahas) need to make this function to actually mimic the navbar functionality
```

Comment:
can you write up a quick ticket for this todo please?

Reply:
Just did 👍  

https://github.com/influxdata/ui/issues/2548